cpp/arcticdb/entity/metrics.hpp

```diff
@@ -133,9 +133,8 @@ class PrometheusInstance {
             return;
         }
 
-        metrics.erase({name, labels});
-        metrics_family.erase(it);
```

Comment:
It looks like the remove operation also removes metrics-family though metrics-family might have additional metrics